src/org/opendatakit/briefcase/model/FormKey.java

```diff
@@ -0,0 +1,96 @@
+package org.opendatakit.briefcase.model;
+
+import static org.opendatakit.briefcase.model.AsJson.getJson;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.util.Objects;
+import java.util.Optional;
+import org.opendatakit.briefcase.reused.BriefcaseException;
+
+/**
+ * This class identifies univocally a local or remote form.
```

Comment:
I didn’t go to university, so I didn’t know this word.

https://ell.stackexchange.com/questions/154036/the-usage-of-univocal

“Univocal is an apt word for your context, but it is a word used rarely by those without university education. So, depending on your target audience, it may be a very good word, or you may wish to simplify[.]”

Reply:
Huh, I didn't expect that. "Unívoco" is not plain Spanish but you can hear it a lot in computer science chatter.

What adverb would you use to say that "this thing identifies something, and only _that particular something_, without leaving room for interpretation"?

Reply:
unambiguously